topology/California Institute of Technology/Caltech CMS Tier2/CIT_CMS_T2.yaml

```diff
@@ -210,6 +155,33 @@ Resources:
         Description: Generic squid service
     VOOwnership:
       CMS: 100
+  CIT_CMS_T2_2_Squid:
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: 6895284f54fb41f9daafc4b6d0a7ea42c57e31be
+          Name: Justas Balcas
+      Resource Report Contact:
+        Primary:
+          ID: 6895284f54fb41f9daafc4b6d0a7ea42c57e31be
+          Name: Justas Balcas
+      Security Contact:
+        Primary:
+          ID: 6895284f54fb41f9daafc4b6d0a7ea42c57e31be
+          Name: Justas Balcas
+      Submitter Contact:
+        Primary:
+          ID: 78c0cedf2bc566b31787f7d831a76ea25b095654
```

Comment:
I don't think this is currently possible since we don't publicly expose the contact IDs @matyasselmeci

Reply:
I think it would be all right to modify the topology webapp to expose non-private information about contacts -- by which I'm thinking the ID and anything outside of the ContactInformation block of <https://topology.opensciencegrid.org/miscuser/xml>.

We could either add an unauth version of miscuser/xml, or add a new interface that lets us do the lookup (and gives results in a more palatable format such as JSON), or both. Should be doable in an afternoon.